gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
@tarungarg546 Yeps, for testing the code.

Don't have experience with Travis but nothing wrong in learning new things. :smile: Will look into it and share more information. Creating an issue for this. :blush: 

Also merging this.